R/dbplyr.R

```diff
@@ -0,0 +1,282 @@
+#' @include DBI.R
+NULL
+
+temporaryErrorMessage <- paste0("You cannot use `temporary = TRUE`",
```

Comment:
I think this would be slightly better as `checkTemporary()`, i.e. including the boilerplate `if (temporary) stop(temporaryErrorMessage)`